test/tedious/tedious.js

```diff
@@ -37,7 +37,9 @@ describe('tedious', () => {
         req.query(require('fs').readFileSync(join(__dirname, '../prepare.sql'), 'utf8'), err => {
           if (err) return done(err)
 
-          sql.close(done)
+          sql.close(err => {
+            done(err)
+          })
```

Comment:
please undo this change

Reply:
Forgive me, I have never participated in any GitHub collaborative development before. Thanks for your understanding.